addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/station/NAModule1Handler.java

```diff
@@ -37,12 +37,18 @@ protected State getNAThingProperty(String channelId) {
         if (module != null) {
             NADashboardData dashboardData = module.getDashboardData();
             switch (channelId) {
-                case CHANNEL_PRESS_TREND:
```

Comment:
Yes, this was an error, there is no pressure sensor on the external module but it was missing on the station itself.